components/dashboard/src/components/start-workspace.tsx

```diff
@@ -354,19 +389,27 @@ export class StartWorkspace extends React.Component<StartWorkspaceProps, StartWo
         if (isError) {
             // If docker build failed
             if (isBuildingWorkspaceImage) {
-              message = <div className="message action">
-                  <Button className='button' variant='outlined' color='secondary' onClick={() => {
-                    this.startWorkspace(this.props.workspaceId, true, true);
-                  }}>Start with Default Docker Image</Button>
-              </div>;
+                message = <div className="message action">
+                    <Button className='button' variant='outlined' color='secondary' onClick={() => {
+                        this.startWorkspace(this.props.workspaceId, true, true);
+                    }}>Start with Default Docker Image</Button>
+                </div>;
             } else {
-              message = <div className="message action">
-                  <Button className='button' variant='outlined' color='secondary' onClick={() => this.redirectToDashboard()}>Go to Workspaces</Button>
-              </div>;
+                message = <div className="message action">
+                    <Button className='button' variant='outlined' color='secondary' onClick={() => this.redirectToDashboard()}>Go to Workspaces</Button>
+                </div>;
             }
         }
         if (this.state && this.state.workspaceInstance && this.state.workspaceInstance.status.phase == 'running') {
-            if (this.state.inTheiaAlready) {
+            if (this.state.remainingUsageHours !== undefined && this.state.remainingUsageHours <= 0) {
+                errorMessage = 'You have run out of Gitpod Hours.';
+                message = <div className='message'>
+                    <div>Gitpod Credit Alert</div>
+                    <Button className='button' variant='outlined' color='secondary' onClick={() =>
+                        window.open(new GitpodHostUrl(window.location.toString()).asUpgradeSubscription().toString(), '_blank')
+                    }>Upgrade Subscription</Button>
+                </div>;
+            } else if (this.state.inTheiaAlready) {
                 message = <div className='message'></div>;
             } else {
                 this.ensureWorkspaceAuth(this.state.workspaceInstance.id)
```

Comment:
in stopping and stopped we don'T want to see productivity tips.
I.e. see https://github.com/gitpod-io/gitpod/pull/1907/files#diff-9b675c748d29d1be3fb144e36f5b22ceR421
we should